rules/order.js

```diff
@@ -59,18 +107,59 @@ module.exports = {
       order: 'strict',
       properties: [
         'background',
+        'background-attachment',
+        'background-clip',
         'background-color',
-        'background-size',
-        'background-position',
+        'background-image',
         'background-repeat',
+        'background-position',
+        'background-size',
```

Comment:
ooh! Good thinking: https://developer.mozilla.org/en/docs/Web/CSS/background

According to that, we missed `background-origin` (should be directly after `-clip`) and `-position` and `-repeat` are backwards.